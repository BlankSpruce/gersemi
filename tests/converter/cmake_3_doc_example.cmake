../inputs/cmake_3_doc_example.cmake