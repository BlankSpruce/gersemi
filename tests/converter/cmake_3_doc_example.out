../parser/cmake_3_doc_example.cmake